samples/README.md

```diff
@@ -25,43 +49,96 @@ exporters:
     - name: LOG_LEVEL
       value: DEBUG
     - name: NAMESPACES
-      value: basic-nginx-prod
+      value: mongo-persistent
     source_ref: master
-    source_url: https://github.com/konveyor/pelorus.git
+    source_url: https://github.com/your_org/pelorus.git
   - app_name: committime-generic
     env_from_secrets:
     - github-secret
-    - mongodb-build-webhook
     source_context_dir: exporters/
     extraEnv:
     - name: APP_FILE
       value: committime/app.py
     - name: LOG_LEVEL
       value: DEBUG
     source_ref: master
-    source_url: https://github.com/konveyor/pelorus.git
+    source_url: https://github.com/your_org/pelorus.git
+```
+3. Documentation regarding values.yaml can be found [here](https://github.com/konveyor/pelorus/blob/master/docs/Configuration.md).  
+  Apply the updated values for Pelorus by executing:
+```
+helm upgrade pelorus charts/pelorus --namespace pelorus --values /var/tmp/values.yaml
+```
+***Note*** pause to allow the committime exporter pod to build and deploy.
+
+Pelorus should now be installed and configured to measure the todolist sample app. We'll have to deploy the sample application to view measurements from Pelorus's Grafana dashboard.
+
+## Todolist sample application install steps
+
+1. git clone the forked copy of the 
+```
+git clone https://github.com/your_org/mig-demo-apps.git
+```
+2. Install the todolist-mongo-go sample application
+```
+cd mig-demo-apps/apps/todolist-mongo-go
+sed -i 's/your_org/YOUR_REAL_GITHUB_FORK_ORG/g' mongo-persistent.yaml
+oc create -f mongo-persistent.yaml
+```
+
+The todolist application and mongo database should now build and deploy into the mongo-persistent namespace.
+
+3. Check the build
+```
+oc get all -n mongo-persistent
```

Comment:
It took a while to get the metric being pulled by exporter, maybe it's worth mentioning here.